core/src/operations/mod.rs

```diff
@@ -1,131 +1,242 @@
 use super::Felt;
 use core::fmt;
 mod decorators;
-use alloc::string::ToString;
 pub use decorators::{
     AdviceInjector, AssemblyOp, DebugOptions, Decorator, DecoratorIterator, DecoratorList,
     SignatureKind,
 };
-use miden_crypto::ZERO;
 use winter_utils::{ByteReader, ByteWriter, Deserializable, DeserializationError, Serializable};
 
+// OPERATIONS OP CODES
+// ================================================================================================
+
+use opcode_constants::*;
+pub(super) mod opcode_constants {
+    pub const OPCODE_NOOP: u8 = 0b0000_0000;
+    pub const OPCODE_EQZ: u8 = 0b0000_0001;
+    pub const OPCODE_NEG: u8 = 0b0000_0010;
```

Comment:
nit: I would format this so that all opcode values are aligned so that it would be easy to visually scan through them. `rustfmt` will complain, but I think it is OK to turn it off for this section.

Reply:
fixed